code/lib/types/src/modules/csf.ts

```diff
@@ -1,7 +1,111 @@
-import type { ViewMode as ViewModeBase } from '@storybook/csf';
+import type {
+  ViewMode as ViewModeBase,
+  AnnotatedStoryFn,
+  Args,
+  ArgsEnhancer,
+  ArgsFromMeta,
+  ArgsStoryFn,
+  ArgTypes,
+  ArgTypesEnhancer,
+  BaseAnnotations,
+  ComponentAnnotations,
+  ComponentId,
+  ComponentTitle,
+  Conditional,
+  DecoratorApplicator,
+  DecoratorFunction,
+  Globals,
+  GlobalTypes,
+  IncludeExcludeOptions,
+  InputType,
+  LegacyAnnotatedStoryFn,
+  LegacyStoryAnnotationsOrFn,
+  LegacyStoryFn,
+  LoaderFunction,
+  Parameters,
+  PartialStoryFn,
+  PlayFunction,
+  PlayFunctionContext,
+  Renderer,
+  SBArrayType,
+  SBEnumType,
+  SBIntersectionType,
+  SBObjectType,
+  SBOtherType,
+  SBScalarType,
+  SBType,
+  SBUnionType,
+  SeparatorOptions,
+  StepFunction,
+  StepLabel,
+  StepRunner,
+  StoryAnnotations,
+  StoryAnnotationsOrFn,
+  StoryContext,
+  StoryContextForEnhancers,
+  StoryContextForLoaders,
+  StoryContextUpdate,
+  StoryFn,
+  StoryId,
+  StoryIdentifier,
+  StoryKind,
+  StoryName,
+  StrictArgs,
+  StrictArgTypes,
+  StrictGlobalTypes,
+  StrictInputType,
+  Tag,
+} from '@storybook/csf';
 
 import type { Addon_OptionsParameter } from './addons';
 
+export type ControlType =
+  | 'boolean'
+  | 'color'
+  | 'date'
+  | 'number'
+  | 'range'
+  | 'object'
+  | 'text'
+  | 'radio'
+  | 'inline-radio'
+  | 'check'
+  | 'inline-check'
+  | 'select'
+  | 'file'
+  | 'multi-select';
+
+export type InputTypeOverride = InputType & {
```

Comment:
Calling it `InputTypeOverride` feels like a temporary name. It should maybe be changed.